internal/pkg/cli/task_run.go

```diff
@@ -157,22 +161,39 @@ func newTaskRunOpts(vars runTaskVars) (*runTaskOpts, error) {
 	return &opts, nil
 }
 
-func (o *runTaskOpts) configureRunner() taskRunner {
+func (o *runTaskOpts) configureRunner() (taskRunner, error) {
 	vpcGetter := ec2.New(o.sess)
 	ecsService := awsecs.New(o.sess)
 
 	if o.env != "" {
+		deployStore, err := deploy.NewStore(o.store)
+		if err != nil {
+			return nil, fmt.Errorf("connect to copilot deploy store: %w", err)
+		}
+
+		d, err := describe.NewEnvDescriber(describe.NewEnvDescriberConfig{
+			App:             o.appName,
+			Env:             o.env,
+			ConfigStore:     o.store,
+			DeployStore:     deployStore,
+			EnableResources: false, // we don't need show resources
+		})
+		if err != nil {
+			return nil, fmt.Errorf("creating describer for environment %s in application %s: %w", o.env, o.appName, err)
```

Comment:
nit: can we use present tense to be consistent with other error messages? Thanks!!
```suggestion
			return nil, fmt.Errorf("create describer for environment %s in application %s: %w", o.env, o.appName, err)
```